docs/index.html

```diff
@@ -37,6 +37,7 @@ <h2 class="title">A 1x Engineer...</h2>
         <li>Pushes to production on a Friday.</li>
         <li>Doesn't need to be passionate about the code they write or the problems they solve, but may be.</li>
         <li>Are willing and able to collaborate with others.</li>
+        <li>Doesn't act surprised when someone doesn’t know something.</li>
```

Comment:
'Does not feel better or worse than someone else for knowing/not knowing something.'

Reply:
The idea was not really about describing how someone feels, more about the reaction given to others which makes them feel worse about not knowing something (it's also a part of Recurse Center social rules https://www.recurse.com/social-rules). Maybe there is a way to word it differently.